deployments/installing-a-package-with-params/Deployment.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Name": "Deployment with Parameters",
+  "ScheduledDateTimeUtc": null,
+  "LastScheduledDateTimeUtc": null,
+  "RepeatPeriod": 0,
+  "ExecutionTimeoutInSeconds": 14400,
+  "RequiresApproval": false,
+  "DeploymentSteps": [
+    {
+      "Name": "Deploy Python to Agent Directory",
+      "IsPrivileged": true,
+      "PlanOrder": 1,
+      "Script": "choco install python312 --params=\"/InstallDir:C:\\BuildAgent\\tools\\python312\"\nexit $LASTEXITCODE",
```

Comment:
Should we be applying [scripting best practices](https://docs.chocolatey.org/en-us/choco/commands/#scripting-integration-best-practices-style-guide) here?

Reply:
Which one are you meaning?

Reply:
Double quoting of option values `"''"`